README.md

```diff
@@ -22,10 +22,11 @@ It's very easy to get up and running with WebCM using `npx`!
 
 You might want to make WebCM load a locally developed Managed Component.
 
-1. Place your Managed Component inside a `components` directory next to your config file
```

Comment:
should we keep the information about the config-file based way of loading WebCM in the README somewhere? It still has its use cases I think (e.g. I want to load two components, I want to try out permissions).